examples/ping_pong.rs

```diff
@@ -86,27 +87,34 @@ async fn remote_handler(req: Request<Body>) -> Result<Response<Body>, hyper::Err
 
 #[tokio::main]
 async fn main() -> Result<(), Error> {
-    /*env_logger::Builder::new()
-    .format(|buf, record| {
-        writeln!(
-            buf,
-            "{}:{} [{}] {} - {}",
-            record.file().unwrap_or("unknown"),
-            record.line().unwrap_or(0),
-            record.level(),
-            chrono::Local::now().format("%H:%M:%S.%6f"),
-            record.args()
-        )
-    })
-    .filter(None, log::LevelFilter::Trace)
-    .init();*/
+    env_logger::init();
```

Comment:
could we follow the example: https://github.com/webrtc-rs/examples/blob/main/examples/reflect/reflect.rs#L49-L65, by using "--debug" to enable the log?

Reply:
Sure thing

Reply:
since i deliberately designed it this way, i don't find it confusing ;)